### PR TITLE
[PK-217] Logging integration based on SLF4J

### PR DESCRIPTION
**Pull request by packpaul**
Sysouts replaced by LOGGER logging.

## Overview

Please describe your changes here.

---

### Checklist

- [ ] Change is covered by automated tests.
- [ ] JavaDoc is provided for all new/changed public APIs.
- [ ] User Guide is updated.
- [ ] CI builds pass.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -58,6 +58,7 @@
         <!--dependency versions -->
         <checkstyle.version>8.18</checkstyle.version>
         <gson.version>2.8.2</gson.version>
+        <slf4j.version>1.7.25</slf4j.version>
```
Comment (inponomarev) [inline pom.xml]: 1.7.26 is the latest stable version 😃

File: celesta-sql/src/test/java/ru/curs/celesta/score/ScoreTest.java [hunk 2]
```diff
@@ -323,11 +327,11 @@ public void saveTest() throws ParseException, IOException {
         PrintWriter bw = new PrintWriter(sw);
         new CelestaSerializer(bw).save(t);
         bw.flush();
-        // System.out.println(sw);
+        // LOGGER.info("{}", sw);
```
Comment (inponomarev) [inline celesta-sql/src/test/java/ru/curs/celesta/score/ScoreTest.java]: well... do we need this? ) in fact this commented-out lines are just helpers for debugging

recently I learnt that one can set `System.out.println`s  in breakpoint properties in IDEA, without adding `println`s to the code (maybe Eclipse also supports something like this)

so maybe just swipe out the junk from code here....?

**Review by inponomarev — CHANGES_REQUESTED** [review]
Alright, to fully finalize the adoption of logger I think we need to add a check to checkstyle rules that prohibits using of `System.(out|err).print(ln|f)` in the code, as [this answer](https://stackoverflow.com/a/19362165/930779) suggests. 

This will prevent us from using `System.out` accidentally later.

File: pom.xml (hunk 1 above)
Comment (packpaul) [reply pom.xml]: So it should have evolved in the meantime.

File: celesta-sql/src/test/java/ru/curs/celesta/score/ScoreTest.java (hunk 2 above)
Comment (packpaul) [reply celesta-sql/src/test/java/ru/curs/celesta/score/ScoreTest.java]: Done.

**Comment by inponomarev** [comment]
Alright. In fact, Checkstyle has suppressions.xml and suppression filters.

With suppressions.xml you can setup the suppression of certain rules for certain files/packages/etc.  And if you setup a filter, you can use `//CHECKSTYLE:OFF`-`//CHECKSTYLE:ON`-like comments in code to turn off/on the checks.

But in this case I think it's too much monkey business. I approve the current version and I think [the time has come](https://jira.curs.ru/browse/PK-230) to move `dbschemasync` into its own project. We already have another satellite project -- [celestadoc](https://github.com/CourseOrchestra/celestadoc), so I think it's OK to extract the `dbschemasync` now.

**Review by inponomarev — APPROVED** [review]

**Pull request merged**